jormungandr-lib/Cargo.toml

```diff
@@ -22,8 +22,8 @@ rand_chacha = "0.3"
 chrono = { version = "0.4", features = ["serde"] }
 humantime = "2.0"
 thiserror = "1.0"
-poldercast = "0.14"
-multiaddr = { package = "parity-multiaddr", version = "0.9" }
+poldercast = { git = "https://github.com/Zeegomo/poldercast", branch = "master" }
```

Comment:
Do we still need it in chain-libs? IIRC multiaddr should be sufficient. It is better to keep deps of `jormungandr-lib` minimal since it is used in places like `jcli`.

Reply:
Well spotted!